subprojects/docs/src/docs/userguide/javaLibraryPlugin.xml

```diff
@@ -87,29 +87,24 @@
             <listitem>types exclusively found in internal classes (future versions of Gradle will let you declare which packages belong to the public API)</listitem>
```

Comment:
I don't think it makes sense. Projects have internal classes, and we should not force users to separate what they want to expose from what they don't want to expose. It's orthogonal to this API/impl separation.